### PR TITLE
Refactored CSS stylesheets to CSS modules.

### PR DESCRIPTION
**Pull request by asownder95**

### CODE REVIEW

File: client/src/components/Menu.jsx [hunk 1]
```diff
@@ -17,6 +19,7 @@ class Menu extends React.Component {
     this.subMenusList = [];
     this.getMenuObj();
     this.handleScroll();
+    console.log(filterTitle);
```
Comment (aliu730) [inline client/src/components/Menu.jsx:22]: remove console

File: client/src/components/Menu.jsx [hunk 2]
```diff
@@ -55,7 +58,7 @@ class Menu extends React.Component {
 
   handleFilterBtnClick(event) {
     let filters = Object.assign({}, this.state.selectedFilters);
-    let targetFilter = event.target.parentElement.querySelector('.filterTitle');
+    let targetFilter = event.target.parentElement.querySelector(`.${filterTitle}`);
```
Comment (aliu730) [inline client/src/components/Menu.jsx:61]: is filterTitle a variable? Why does it need a ${}?

File: client/src/components/SubMenuSectionEntry.jsx [hunk 3]
```diff
@@ -16,13 +17,12 @@ class SubMenuSectionEntry extends React.Component {
     let image;
```
Comment (aliu730) [inline client/src/components/SubMenuSectionEntry.jsx:17]: remove comment

File: public/index.html [hunk 4]
```diff
@@ -8,7 +8,7 @@
     <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="./styles/main.css">
+    <!-- <link rel="stylesheet" href="./styles/main.css"> -->
```
Comment (aliu730) [inline public/index.html:11]: comment remove

**Review by aliu730 — APPROVED** [review]

File: client/src/components/Menu.jsx (hunk 2 above)
Comment (asownder95) [reply client/src/components/Menu.jsx:61]: Destructured another CSS style module to be able to toggle a class on a child component. That's what filterTitle is.

**Pull request merged**